src/NuGet.Core/NuGet.CommandLine.XPlat/Commands/Signing/TrustedSignersCommand.cs

```diff
@@ -23,107 +22,305 @@ internal static class TrustedSignersCommand
+                        Strings.Option_ConfigFile,
+                        CommandOptionType.SingleValue);
+
+                    syncCommand.HelpOption(XPlatUtility.HelpOption);
+
+                    CommandOption verbosity = syncCommand.VerbosityOption(); ;
+
+                    CommandArgument name = syncCommand.Argument("<NAME>",
+                                               Strings.TrustedSignerNameExists);
+
+                    syncCommand.OnExecute(async () =>
+                    {
+                        return await ExecuteCommand(TrustCommand.Sync, algorithm: null, allowUntrustedRootOption: false, owners: null, verbosity, configFile, getLogger, setLogLevel, name: name.Value);
+                    });
+                });
+
+                trustedSignersCmd.Command("remove", (syncCommand) =>
+                {
+                    syncCommand.Description = Strings.TrustRemoveCommandDescription;
+                    CommandOption configFile = syncCommand.Option(
+                        "--configfile",
+                        Strings.Option_ConfigFile,
+                        CommandOptionType.SingleValue);
+
+                    syncCommand.HelpOption(XPlatUtility.HelpOption);
+
+                    CommandOption verbosity = syncCommand.VerbosityOption();
+
+                    CommandArgument name = syncCommand.Argument("<NAME>",
+                                               Strings.TrustedSignerNameToRemove);
+
+                    syncCommand.OnExecute(async () =>
+                    {
+                        return await ExecuteCommand(TrustCommand.Remove, algorithm: null, allowUntrustedRootOption: false, owners: null, verbosity, configFile, getLogger, setLogLevel, name: name.Value);
+                    });
+                });
+
+                trustedSignersCmd.Command("author", (authorCommand) =>
+                {
+                    authorCommand.Description = Strings.TrustAuthorCommandDescription;
+
+                    CommandOption allowUntrustedRootOption = authorCommand.Option(
+                                                "--allow-untrusted-root",
+                                                Strings.TrustCommandAllowUntrustedRoot,
+                                                CommandOptionType.NoValue);
+
+                    CommandOption configFile = authorCommand.Option(
+                        "--configfile",
+                        Strings.Option_ConfigFile,
+                        CommandOptionType.SingleValue);
+
+                    authorCommand.HelpOption(XPlatUtility.HelpOption);
 
-                CommandOption configFile = trustedSignersCmd.Option(
+                    CommandOption verbosity = authorCommand.VerbosityOption();
+
+                    CommandArgument name = authorCommand.Argument("<NAME>",
+                                               Strings.TrustedSignerNameToAdd);
+                    CommandArgument package = authorCommand.Argument("<PACKAGE>",
```

Comment:
As per [spec](https://github.com/NuGet/Home/blob/b30a8ff077272045224829c6772327b724ebed35/proposed/2021/DotnetNugetTrustedSigners.md#note), `For trust author and trust repository, <package> may be a glob/wildcard. This must be consistent across platforms, so the Windows version must expand these wildcards when it receives them as arguments.`. There was a similar requirement for `dotnet nuget verify` command, hence we passed `multipleValues: true`.

https://github.com/NuGet/NuGet.Client/blob/966b1b21eec28834365a8604744dab682c8326ea/src/NuGet.Core/NuGet.CommandLine.XPlat/Commands/Signing/VerifyCommand.cs#L24-L27

Reply:
We had discussion on this one and decided not to in favor in security concern. https://github.com/NuGet/Home/issues/10647#issuecomment-801523491

Reply:
`Verify` command doesn't mutate anything so no security concern.